nextjs-client/src/hooks/useScrollPosition.js

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react';
+
+export const useScrollPosition = () => {
+  const [scrollPosition, setScrollPosition] = useState(0);
+
+  useEffect(() => {
+    const updatePosition = () => {
+      setScrollPosition(window.pageYOffset);
+    };
+
+    window.addEventListener('scroll', updatePosition);
+
+    updatePosition();
```

Comment:
Just a question, do we need this call here? isn't `updatePosition()` gets called as a callback in line 11?